rubi/rubi/contracts/erc20.py

```diff
@@ -404,3 +404,14 @@ def to_integer(self, number: Decimal) -> int:
             return 0
         else:
             return int(number * (10 ** self.decimal))
+
+    def max_approval_amount(self) -> Decimal:
+        """return the max uint256 token approval amount. Note: this is not very secure and if you give this approval you
+        should revoke it when you are done!
+
+        :return: the max approval amount of a uint256 token in Decimal representation
+        :rtype: Decimal
+        """
+        max_uint256 = "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
```

Comment:
this totally works, was wondering if there is a reason we didn't do something like: 
`return self.to_decimal(number=2**256 - 1)`